src/VisualStudio/Core/Def/Implementation/CodeLens/VisualStudioCodeLensReferencesService.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeLens;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeLens
+{
+    [ExportWorkspaceService(typeof(ICodeLensReferencesService), layer: ServiceLayer.Host), Shared]
+    internal sealed class VisualStudioCodeLensReferencesService : ICodeLensReferencesService
+    {
+        public async Task<ReferenceCount> GetReferenceCountAsync(Solution solution, DocumentId documentId, SyntaxNode syntaxNode, int maxSearchResults,
+            CancellationToken cancellationToken)
+        {
+            var remoteService = solution.Workspace.Services.GetService<IRemoteCodeLensReferencesService>();
```

Comment:
Gotcha, I was confused on why that was done and just blindly copied it. I'll remove it.